gui/default/syncthing/core/syncthingController.js

```diff
@@ -51,6 +51,9 @@ angular.module('syncthing.core')
         $scope.failedPageSize = 10;
         $scope.scanProgress = {};
         $scope.themes = [];
+        $scope.globalChangeEvents = {};
+        $scope.findDevice = {};
+        $scope.deviceName = {};
```

Comment:
I'll have to look into this.  I believe I just need to delete those as I decided not to use them anyway.